com.unity.perception/Runtime/GroundTruth/Labelers/KeypointLabeler.cs

```diff
@@ -491,6 +736,15 @@ private void InitKeypoint(Vector3 position, Keypoint[] keypoints, int idx)
                 keypoints[idx].y = loc.y;
                 keypoints[idx].state = 2;
             }
+
+            //TODO: move this code
```

Comment:
:shrug: maybe just remove this comment, I'm not exactly sure where you would like to move this code to

Reply:
Done.